nano/node/voting.cpp

```diff
@@ -362,7 +362,9 @@ void nano::vote_generator::vote (std::vector<nano::block_hash> const & hashes_a,
 	std::vector<std::shared_ptr<nano::vote>> votes_l;
 	wallets.foreach_representative ([this, &hashes_a, &votes_l] (nano::public_key const & pub_a, nano::raw_key const & prv_a) {
 		auto timestamp = this->is_final ? nano::vote::timestamp_max : nano::milliseconds_since_epoch ();
-		votes_l.emplace_back (std::make_shared<nano::vote> (pub_a, prv_a, timestamp, hashes_a));
+		timestamp &= nano::vote::timestamp_mask;
+		uint8_t duration = this->is_final ? nano::vote::duration_max : /*8192ms*/ 0x9;
```

Comment:
Should the default duration value be configurable?

Reply:
No the duration needs to be software controlled.